spec/defines/pip_spec.rb

```diff
@@ -144,6 +151,34 @@
       end
     end
   end
+
+  context 'on Debian OS with pip_version 20.3.4' do
+    let :facts do
+      {
+        id: 'root',
+        kernel: 'Linux',
+        lsbdistcodename: 'squeeze',
+        os: {
```

Comment:
could you instead mock an OS that we currently support? (we should really rewrite the whole file to use rspec-puppet-facts...)

Reply:
Sure! I'll see what I can do after this PR :-)